qualtran/resource_counting/classify_bloqs.py

```diff
@@ -244,10 +244,12 @@ def bloq_is_rotation(b: Bloq) -> bool:
     if isinstance(b, Controlled):
```

Comment:
This branch captures any `CtrlSpec` right? For non-trivial ctrl-spec, shouldn't we error out? (or use the and-ladder, but perhaps for later) #1290

Reply:
yes, that is an issue; but not a _new_ issue introduced by this PR. Let me see if I can sneak a quick fix in or if it'll be a whole thing; in which case, I will defer to another PR

Reply:
I could -- there aren't any places in the unit tests or bloq examples that relies on `Controlled(rot, #cvs>1)`